docs/kubernetes/provisioning/capi/README.md

```diff
@@ -0,0 +1,8 @@
+# Cluster API (CAPI)
```

Comment:
@jakefhyde Just an FYI about these docs, incase you want to review / expand on them in the future. It's Arvinds last day so we've gone ahead and merged them, but if there are any corrections / updates you'd like to make (or if you want to move these to a different repo), lmk